cartoframes/data/observatory/catalog/entity.py

```diff
@@ -126,35 +127,21 @@ def _download(self, credentials, file_path=None, limit=None, order_by=None):
         if not self._is_available_in('bq'):
             raise DOError('{} is not ready for Download. Please, contact us for more information.'.format(self))
 
-        bq_client = _get_bigquery_client(credentials)
```

Comment:
You can probably do a cleanup of the unused stuff

Reply:
👌